cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -511,6 +516,14 @@ func to(ctx context.Context, client *sqlclient.Client) (migrate.StateReader, err
 	}
 }
 
+func parseHCLBytes(b []byte) (*hclparse.Parser, error) {
```

Comment:
why do you add there the type to the function? 
```suggestion
func parseHCL(b []byte) (*hclparse.Parser, error) {
```